cardanotiddies

```diff
@@ -0,0 +1,14 @@
+{
+    "project": "Cardano Tiddies",
+    "tags": [
+        "titty"
+        "tiddy"
+        "cardano"
+        "ADA
+        
+    ],
+    "policies": [
+       69bdc2bff8f9d1f6a4e6b990e9c9845286c8461722850f6ac24c14bb              
```

Comment:
Add the speech marks at the beginning and the end of the Policy ID in the code.

Reply:
> Add the speech marks at the beginning and the end of the Policy ID in the code.

Hi. was able to edit. hope it reflects on your end